File: Sorting/bubble_sort.py
def bubble_sort(a):
    for i in range(len(a)-1):
        for j in range(i+1,len(a)):
            if a[j] <a[i]:
                a[i],a[j]=a[j],a[i]
    print(a)
a=[1,5,7,2,3,6,10,8,12,5,100,34,64,7,11]
bubble_sort(a)

File: Sorting/bubble_sort.go
package main

import "fmt"

func bubble_sort(a []int) {
	for i := 0; i < len(a)-1; i++ {
		for j := i + 1; j < len(a); j++ {
			if a[j] < a[i] {
				a[i],a[j]=a[j],a[i]
			}
		}
		//a[i], a[min] = a[min], a[i]
	}
	fmt.Println(a)
}
func main() {
	a := []int{4, 3, 2, 40,53, 1, 10, 7, 100, 23, 27}
	bubble_sort(a)
}
